src/Template/Element/Form/relations.twig

```diff
@@ -7,11 +7,11 @@
 {% if relations %}
 <section class="relations">
 
-    {% for relationName in relations %}
+    {% for relationName in relations if relationName != 'roles' and relationName != 'translations' %}
```

Comment:
we need to find a better rule to distiguish between `dynamic object relations` and `static object/resources` relations
best solution is to read this info from API

for now we may want to discard only `roles` or have two different arrays from the controller for `dynamic` and `static` relations